### PR TITLE
NXP-23101: upgrade to Polymer 2 legacy API

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: elements/document/file/nuxeo-file-metadata-layout.html [hunk 1]
```diff
@@ -22,8 +22,8 @@
 -->
 
 <dom-module id="nuxeo-file-metadata-layout">
-
   <template>
+    <style include="nuxeo-styles"></style>
```
Comment (nelsonsilva) [inline elements/document/file/nuxeo-file-metadata-layout.html]: Unnecessary, layout behavior stamps these in light DOM

File: elements/document/note/nuxeo-note-create-layout.html [hunk 2]
```diff
@@ -94,6 +97,10 @@
           {id: 'text/xml', label: this.formatMimeType('text/xml')},
           {id: 'text/x-web-markdown', label: this.formatMimeType('text/x-web-markdown')}
         ]
+      },
+
+      _documentChanged: function() {
```
Comment (nelsonsilva) [inline elements/document/note/nuxeo-note-create-layout.html]: Intended?

**Comment by guirenard** [comment]
:+1:

**Comment by Gabez0r** [comment]
👍

**Comment by Gabez0r** [comment]
👍

**Comment by adilioaraujo** [comment]
:+1:

**Comment by pncosta** [comment]
:+1:

**Pull request merged**